app/models/venue.rb

```diff
@@ -1,10 +1,24 @@
 class Venue < ActiveRecord::Base
+
+  STATES = ["AL", "AK", "AZ", "AR", "CA", "CO", "CT", "DE", "FL",
+    "GA","HI", "ID", "IL", "IN", "IA", "KS", "KY", "LA", "ME", "MD",
```

Comment:
Align the elements of an array literal if they span more than one line.<br>Space missing after comma.